devices/Adxl343/.gitignore

```diff
@@ -0,0 +1,6 @@
+﻿################################################################################
```

Comment:
please remove this file, we have a general one :-)

Reply:
Sorry.  Forgot.